deployments/operator/webhook/manifests.yaml

```diff
@@ -133,6 +133,7 @@ webhooks:
       name: webhook-service
       namespace: system
       path: /pods-sgx
+  reinvocationPolicy: IfNeeded
```

Comment:
thanks! can you add it to `sgx_admissionwebhook` too?

Reply:
done.

Reply:
one small ask still: can you squash the two commits in one and force push. otherwise LGTM

Reply:
done.